vendor/vendor.json

```diff
@@ -515,10 +515,10 @@
 			"revisionTime": "2016-08-03T19:07:31Z"
 		},
 		{
-			"checksumSHA1": "5vT6l22kAQVXWOT1YI5bhLQ9kBA=",
+			"checksumSHA1": "o2NFiTnCfAMIdWJWIsIeXcoQ1Ws=",
```

Comment:
I patched go-icinga2-api and groups should be optional now, if you use
   {
                        "checksumSHA1": "0FjdzTP/wqt4z636v+L6YHXWMCM=",
                        "path": "github.com/lrsmith/go-icinga2-api/iapi",
                        "revision": "61eb49b68376820a4bf309318033b3c7ce425c87",
                        "revisionTime": "2018-03-09T04:23:47Z",
                        "version": "v0.2.1",
                        "versionExact": "v0.2.1"
                },